.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,57 +1,131 @@
+
+Add only ONE of the following /kind <type> entries.
+
+/kind api-change
+/kind bug
+/kind cleanup
+/kind design
+/kind documentation
+/kind failing-test
+/kind feature
+/kind flake
+-->
+/kind ???
+
+> DISCUSSION REQUIRED
 >
-> /kind api-change
-> /kind bug
-> /kind cleanup
-> /kind design
-> /kind documentation
-> /kind failing-test
-> /kind feature
-> /kind flake
+> Should we include a kind type of `deprecation`? And should we then include a `` ```depracation-notes`` code block to provide specific detailed notes around the depracation.
+> 
+> Should we only allow a single kind to be specified? There are a number of examples where multiple kind values make sense.
 
 **What this PR does / why we need it**:
+<!--
+Provide a description on what this PR does and/or why it is required.
+-->
+Description here ...
 
 **Which issue(s) this PR fixes**:
 <!--
-*Automatically closes linked issue when PR is merged.
-Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
+Add references to issues that are fixed by this PR by using one or more of the following templates:
+
+Fixes #<issue number>
+Fixes <issue url>
+
+NOTE: The linked issue will automatically be closed when the PR is merged.
+
+If the PR is about failing-tests or flakes, please post the related issues/tests in a comment and do not use the Fixes templates above.
+-->
+Fixes #???
+
+**Have you added or run tests for your PR?**
+<!--
+Indicate whether you have added or run the appropriate tests for your PR by using ONE of the following responses:
+
+N/A
+Yes
+No
+
+More details can be found in the Testing Guide:
+https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
 -->
-Fixes #
+No
 
 **Special notes for your reviewer**:
+<!--
+Provide any additional notes that may be helpful to the PR reviewers.
+-->
+None
 
-**Does this PR introduce a user-facing change?**:
+**What user facing changes are introduced by this PR?**:
 <!--
-If no, just write "NONE" in the release-note block below.
-If yes, a release note is required:
-Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".
+Indicate whether there are any user facing changes that should be added to the Release Notes. 
+
+If there are no user facing changes, leave the "None" text in the release-note block below. If your PR does introduce user facing changes, then a release note is required. Add your extended release note in the release-note block below.
+
+If the PR requires additional action from users switching to the new release, start the release-notes block with the text "Action required".
+
+Familiarise yourself with the instructions for writing a release note: 
+https://git.k8s.io/community/contributors/guide/release-notes.md
 -->
 ```release-note
-
+None
```

Comment:
I'd prefer not to default this because we want people to consciously apply the release note. An empty block will lead to `do-not-merge/release-note-label-needed` and will prevent merge but defaulting can automatically merge a PR in case someone forgot to mention the release note.

Reply:
> I'd prefer not to default this because we want people to consciously apply the release note. An empty block will lead to `do-not-merge/release-note-label-needed` and will prevent merge but defaulting can automatically merge a PR in case someone forgot to mention the release note.

Very much this ^^^

Reply:
I added this in since there are a large number of release notes with variations of none, n/a etc. I wanted the default to help those folks. But I totally see both of your concerns now. I'm slowly building up an understanding of the inter-dependent bits here.

I'll remove the default and adjust the comment appropriately.